.changeset/weak-crews-judge.md

```diff
@@ -0,0 +1,80 @@
+---
+'@razorpay/blade': major
+---
+
+feat(Typography): streamline typography scale
+
+> **Warning**
+>
+> Breaking Change!
+> This is a breaking change for typography components and lineHeight scale
```

Comment:
add: we have written codemod to ease this process so please follow the migration guide thoroughly(make this link to the section below)

Reply:
Added